libraries/botbuilder/tests/streaming/mockNetSocket.js

```diff
@@ -0,0 +1,18 @@
+const { STATUS_CODES } = require('http');
+
+class MockNetSocket {
+    constructor(readable = true, writable = true) {
+        this.readable = readable;
+        this.writable = writable;
+    }
+
+    write(response) { }
+
+    destroy(err) { }
```

Comment:
These two methods are for spying on during the authentication portion of `useWebSocket()`. The static method `MockNetSocket.createNonSuccessResponse()` is for testing the 401 scenario in useWebSocket().